File: Java/api/src/main/java/com/fiera/api/dtos/UserDTO.java
package com.fiera.api.dtos;

import javax.persistence.Id;
import javax.persistence.GeneratedValue;
import javax.validation.constraints.NotNull;

import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
public class UserDTO {

    @Id 
    @GeneratedValue
    private Integer UserId;
    @NotNull(message = "Doc Number is required")
    private String DocNumber;
    @NotNull(message = "First Name is required")
    private String FirstName;
    @NotNull(message = "Last Name is required")
    private String LastName;
    private String Email;
    private String Phone;
    private String Address;
}

File: NetCore/FieraServicesWebAPITest/FieraServicesWebAPITest/DTOs/UserDTO.cs
using System.ComponentModel.DataAnnotations;

namespace FieraServicesWebAPITest.DTOs
{
    public class UserDTO
    {
        [Key]
        public int UserId { get; set; }
        [Required]
        public string DocNumber { get; set; }
        [Required]
        public string FirstName { get; set; }
        [Required]
        public string LastName { get; set; }
        public string Email { get; set; }
        public string Phone { get; set; }
        public string Address { get; set; }
    }
}
